### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by Banstein**
- Setup the Project with Webpack.
- Created a simple UI for the project.
- Used ES6 for importing the CSS loader.
- Install babel loader for ES6.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
       "devDependencies": {
```
Comment (mateo951) [inline package-lock.json:17]: All webpack packages are added as devDependencies 🚀

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
+* {
```
Comment (mateo951) [inline src/style.css:1]: Went beyond the project requirements 🤯 Nice job!

**Review by mateo951 — APPROVED** [review]
### Hi @Banstein, 

📝👌 **Your project highlights** 🔥😎 
- General Requirements ✔
- Gitflow is used correctly 🔥
- Webpack is installed perfectly 🙌
- Wireframe design is great 👌

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 2 above)
Comment (Banstein) [reply src/style.css:1]: Thank you!

**Pull request merged**